### DISCUSSION TITLE
Basemap & dsm scale distortion?

### DISCUSSION THREAD

**Discussion by jmeanxt** [domlysz/BlenderGIS#728] [Q&A]
Hi 
Is any on else finding that when adding the srtm to a google aerial under  web medicator coordinates the heigth is correct but the map scales in the x & y , so that on emeter on the aerial is now 1.6.

I first came across this when i could not understand that hills height i know look smaller on the srtm mesh, i measured the height and it was fine, but i just measured the distance between peeks and Thay were 1.6 time further away from each other. 

Could i be doing some thing wrong or is it a known issue?

thanks

**Comment by jmeanxt** [comment]
this is what i am getting, google earth pro on left & blender gis on right, the image has scaled nearly 2 up ion the xy axis on right (blender gis) but not in the z, any one else experance this with Gis plugin ?
![image](https://user-images.githubusercontent.com/52382722/194703758-b2f04584-2e5e-4aac-9b49-62f63b76fbb8.png)

**Comment by jmeanxt** [comment]
think I got my answer here, : https://github.com/domlysz/BlenderGIS/wiki/FAQ#why-my-measurements-on-basemaps-reference-seems-wrong-

**Comment by eobet** [comment]
I've fallen into this same trap, and I wish this add-on would have built in functionality to help fix this known problem.
